django_freeradius/api/urls.py

```diff
@@ -6,4 +6,6 @@
     url(r'^authorize/$', views.authorize, name='authorize'),
     url(r'^postauth/$', views.postauth, name='postauth'),
     url(r'^accounting/$', views.accounting, name='accounting'),
+    url(r'^v1/batchCsv/$', views.batchCsv, name='batchCsv'),
```

Comment:
adding `v1` here is not the right thing to do, we'll have to add it one level app.
also, the camelCase convention is not suited here, therefore please change this line to:

```python
url(r'^batch-csv/$', views.batchCsv, name='batch-csv'),
```